ui/imports/utils/Constants.qml

```diff
@@ -288,6 +291,20 @@ QtObject {
         readonly property int userImageWidth: 40
         readonly property int userImageHeight: 40
         readonly property int titleFontSize: 17
+        readonly property QtObject profileFetching: QtObject {
+            readonly property int    titleFontSize: 22
+            readonly property string titleForSuccess: qsTrId("Profile successfully fetched")
```

Comment:
`qsTr()`, we don't use `qsTrId()` anywhere

Reply:
done